README.md

```diff
@@ -39,6 +40,11 @@ services:
         ENTRYPOINT: ./YOUR_ENTRYPOINT
 ```
 
+By default, the IOC will be built with statically linked libraries. If you want
+to use dynamically linked libraries, you should define the build target as
+`dynamic-link`. This will increase the resulting image size, since unneeded
+dependencies will also be copied.
```

Comment:
"This will increase the resulting image size, since unused dependencies will also be copied."